CommandHelper/LocalPackages/commands/powers/shadowtravel.msa

```diff
@@ -0,0 +1,5 @@
+shadowtravel:/shadowtravel = >>>
+    sudo("/tppos")
```

Comment:
Doesn't do anything. /tppos requires: /tppos (x) (y) (z) (yaw) (pitch)

Better method:

set_ploc(pcursor()) // Sets the players location to the block they are currently looking at. May require minor tweaking to make sure you don't place the player into a block.